crates/db_views/src/private_message_view.rs

```diff
@@ -336,6 +357,39 @@ mod tests {
       .unwrap();
     assert_eq!(timmy_unread_messages, 1);
 
+    // Make sure instance_blocks are working
+    let timmy_blocks_instance_form = InstanceBlockForm {
+      person_id: timmy.id,
+      instance_id: sara.instance_id,
+    };
+
+    let inserted_instance_block = InstanceBlock::block(pool, &timmy_blocks_instance_form)
+      .await
+      .unwrap();
+
+    let expected_instance_block = InstanceBlock {
+      person_id: timmy.id,
+      instance_id: sara.instance_id,
+      published: inserted_instance_block.published,
+    };
+    assert_eq!(expected_instance_block, inserted_instance_block);
+
+    let timmy_messages = PrivateMessageQuery {
+      unread_only: true,
+      creator_id: None,
+      ..Default::default()
+    }
+    .list(pool, timmy.id)
+    .await
+    .unwrap();
+
+    assert_length!(0, &timmy_messages);
+
+    let timmy_unread_messages = PrivateMessageView::get_unread_messages(pool, timmy.id)
+      .await
+      .unwrap();
+    assert_eq!(timmy_unread_messages, 0);
+
```

Comment:
Can you move this into a separate test case? It gets really difficult to fix failures in such long functions.

Reply:
Done